### DISCUSSION TITLE
Add a DJ Role feature.

### DISCUSSION THREAD

**Discussion by DragonZV** [SudhanPlayz/Discord-MusicBot#71] [Ideas]
Add a DJ Role feature.

A DJ role means when the only a specific role can stop and skip music.

**Comment by asavage7** [comment]
I would also like to see this change, it can be frustrating sometimes when everyone can skip or stop music.

**Comment by ItzSudhan** [comment]
What happen if the DJ is offline and the users needed to skip the song

**Comment by asavage7** [comment]
I don't know how easy this would be to implement but what if there was some sort of voting system?

**Comment by DragonZV** [comment]
> What happen if the DJ is offline and the users needed to skip the song

A DJ role. Meaning multiple users can have the role.

**Comment by ItzSudhan** [comment]
Well yea i have added this to v4 to-do list. I will do soon

**Comment by ItzSudhan** [comment]
Discord Music Bot v4 - #72
